cloudslang-entities/src/main/java/io/cloudslang/lang/entities/constants/Regex.java

```diff
@@ -43,6 +43,14 @@
     public static final String GET_REGEX = "get\\((.+)\\)";
     public static final String GET_REGEX_WITH_DEFAULT = "get\\((.+?),(.+?)\\)";
     public static final String CHECK_EMPTY_REGEX = "check_empty\\((.+?),(.+?)\\)";
+    public static final String CS_APPEND_REGEX = "cs_append\\((.+?),(.+?)\\)";
+    public static final String CS_PREPEND_REGEX = "cs_prepend\\((.+?),(.+?)\\)";
+    public static final String CS_EXTRACT_NUMBER_REGEX = "cs_extract_number\\((.+?)(,(.+?))?\\)";
+    public static final String CS_REPLACE_REGEX = "cs_replace\\((.+?),(.+?),(.+?)(,(.+?))?\\)";
+    public static final String CS_ROUND_REGEX = "cs_round\\((.+?)\\)";
+    public static final String CS_SUBSTRING_REGEX = "cs_substring\\((.+?),(.+?)(,(.+?))?\\)";
+    public static final String CS_TO_UPPER_REGEX = "cs_to_upper\\((.+?)\\)";
+    public static final String CS_TO_LOWER_REGEX = "cs_to_lower\\((.+?)\\)";
```

Comment:
python allows spaces between method name and arguments' parentheses (as well after arguments and colon). e.g: `def myFunction    (a, b)     :`

Reply:
The regex should match the function invocation, not definition. I know that the regex is fairly "generous" (that's how the old ones are as well) and I think it's ok. If they are too complex, it may add a performance penalty. But we can discuss.